### PR TITLE
commiting to my pledge

### PR DESCRIPTION
**Pull request by KaranSantra**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "gatsby-image": "^2.4.5",
     "gatsby-plugin-manifest": "^2.4.9",
     "gatsby-plugin-react-helmet": "^3.3.2",
-    "gatsby-plugin-sharp": "^2.6.9",
+    "gatsby-plugin-sharp": "^2.6.42",
```
Comment (Abhishek-kumar09) [inline package.json:22]: please revert all these changes, you should only change you json file. You could revert these by copying the respective files from the repository.
You can see other merged PRs for help.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (KaranSantra) [reply package.json:22]: Thanks will do that. After I make these changes do I need to send another PR?

Comment (KaranSantra) [reply package.json]: @Abhishek-kumar09 I have reverted all these changes in the next commit

**Comment by Abhishek-kumar09** [comment]
@KaranSantra That's nice! please also revert `package-lock.json`

**Comment by KaranSantra** [comment]
@Abhishek-kumar09  reverted package-lock.json

**Comment by KaranSantra** [comment]
Btw can you give me suggestion on how to improve my technical communication skills (like for slack,github etc)

File: src/data/contributors/KaranSantra.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Karan Santra",
+    "skills" : ["Java", "Node JS"],
+    "cause" : "creating oppurtunities for everyone",
+    "image" : "Avatar5",
+    "github" : "https://github.com/KaranSantra",
+    "twitter" : "https://twitter.com/karan_santra",
+    "linkedin" : "https://www.linkedin.com/in/karan-santra-42633511a"
+}
```
Comment (Abhishek-kumar09) [inline src/data/contributors/KaranSantra.json:9]: Here too :)

File: package-lock.json [hunk 3]
```diff
@@ -18428,4 +18428,4 @@
       "integrity": "sha512-V50KMwwzqJV0NpZIZFwfOD5/lyny3WlSzRiXgA0G7VUnRlqttta1L6UQIHzd6EuBY/cHGfwTIck7w1yH6Q5zUw=="
     }
   }
-}
+}
```
Comment (Abhishek-kumar09) [inline package-lock.json:18431]: Please enter a blank line at the end of file, its a convention.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]

**Comment by Abhishek-kumar09** [comment]
Btw can you give me suggestion on how to improve my technical communication skills (like for slack,github etc)
@KaranSantra You are already good at it.
Generally, Just try to be polite, because everyone in the world is once in their learning phase and can do some mistakes that looks stupid to you or etc, but remember that you had also done the same. Else experience will tell you everything. ...and ask anything smallest to biggest things, but that should be to the point.

For technical things google first before personally asking, because the person you are asking will also google the same thing (in case he is unsure about their answer) and this way you will be wasting everyones time and efforts.

File: src/data/contributors/KaranSantra.json (hunk 2 above)
Comment (KaranSantra) [reply src/data/contributors/KaranSantra.json:9]: I do not understand what changes do I need to make in this file.

Comment (KaranSantra) [reply src/data/contributors/KaranSantra.json:9]: ok you want me to add a blank space here as well. Got it.

File: package-lock.json (hunk 3 above)
Comment (KaranSantra) [reply package-lock.json:18431]: I made the changes then made commit and pushed it. Is this okay or I need to create a new pr?

**Comment by KaranSantra** [comment]
@Abhishek-kumar09 Thank you for your reply. This was my first pr and I have learnt so many new things already. Enjoying this journey so far :)

**Pull request merged**